notebooks/official/workbench/spark/spark_ml.ipynb

```diff
@@ -0,0 +1,1623 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #17.    # Since it consumes too much time and computing resources, we will use 10% of the rows for each dataset.
</pre><p>consider adding a comment that this is not standard practice when doing this in a production environment - this is purely for demonstrative purposes</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/838/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>

Reply:
<p>On the flip side, you can (accurately) add a comment that you experimented with different values for this and didn't see much of a difference in model performance but encourage users to try different values for themselves (and add that the runtime of the notebook linearly scales with this)</p>